albumentations/augmentations/functional.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import division
 
 import math
+import random
```

Comment:
you cannot use random here, it leads to errors in case of additional image-type targets